back/app/models/que_job.rb

```diff
@@ -1,5 +1,30 @@
 # frozen_string_literal: true
 
+# == Schema Information
```

Comment:
Not sure where this came from, should I delete it?

Reply:
It's fine to leave it. It should be here.